input/errors.go

```diff
@@ -0,0 +1,30 @@
+package input
+
+import (
+	"github.com/spacemonkeygo/errors"
+)
+
+var Error *errors.ErrorClass = errors.NewClass("InputError") // grouping, do not instantiate
+
+/*
+	Indicates that the input failed to obtain data that correctly
+	matches the hash specifying the input.  This may mean there have
+	been data integrity issues in the storage or transport systems involved.
+*/
+var InputHashMismatchError *errors.ErrorClass = Error.NewClass("InputHashMismatchError")
+
+/*
+	Indicates that the target filesystem (the one given to `Apply`) had some error.
+*/
+var TargetFilesystemUnavailableError *errors.ErrorClass = Error.NewClass("TargetFilesystemUnavailableError")
```

Comment:
Ick, good eye.  I'm still kind of dead reckoning / flying through fog on errors big-picture-wise, and wondering if maybe expressing which relevant filesystem should just be a [DataKey](https://godoc.org/github.com/spacemonkeygo/errors#DataKey) thing?  So that's something to ponder for future error reviews&refactors... but nonunique names is offensive even in the interim.  Fixing in #32.